_skycap/the-basics/concepts-and-terminology.md

```diff
@@ -0,0 +1,118 @@
+
+At Cloud 66 we use [Kubernetes](https://kubernetes.io/) - the market leading solution - to orchestrate containers and applications in clustered environments.
+
+## Pods, nodes and clusters
+
+In Kubernetes a **pod** is the smallest and simplest object in the system. A typical pod takes a container, gives it access to storage and assigns it an IP address. It also applies various policies and settings to it. 
+
+A pod is like a wrapper in which a single, complete instance of an application (such as a microservice) runs. Pods can, and often do, incorporate multiple containers. This is usually done when the containers need to access a shared resource.
+
+A **node** is a server or “worker machine” in a Kubernetes **cluster**. It can be real or virtual. Each node includes the services required to host **pods**. Each node can host multiple pods. 
+
+It’s important to understand that Kubernetes itself does not create or manage the underlying worker machines - it merely manages the use of the resources of those machines. 
+
+A **cluster** is a collection of **nodes** (and the **pods** running on those nodes) that is managed by the Kubernetes platform. Clusters are the largest and most complex object in Kubernetes. Clusters direct and manage the behaviour of the underlying pods, as well as shared resources like networking and storage.
+Service Networking
+
+In containerized applications, the different containers communicate with each other via an internal, privately routed network. By default Docker relies on host-private networking for this communications, so all containers must be hosted in the same environment (node). 
+
+It is possible for containers to communicate across nodes, but this relies on configuring custom ports and IP routes. This is error and failure prone, particularly if nodes are frequently scaled up or down.
+
+Kubernetes simplifies this process through a feature called a “Kubernetes Service” (or just “service” for short). A service manages the assignment of IP addresses and ports, based on whatever parameters you define. 
+
+## Service Networking
+
+It's likely that some services inside your application will need to respond to queries from the public internet.
+
+For a service to be available to anyone outside the container, we need to bridge it from inside to outside of the container.
+
+For example a container with a web application might be set to listen on port 3000 within the internal (Kubernetes) cluster network, and that port could then be exposed to port 80 on the public network attached to the cluster. 
+
+This is not limited to HTTP or web traffic. The same concepts apply if a container serves non-HTTP traffic (like web sockets, DB containers or custom TCP / UDP traffic).
+
+***Note***
```

Comment:
Is *note* the correct sub-title here? Maybe `class="notice"` is sufficient to call out this content? There is a styleguide that may be helpful for content design and typography. http://cloud66.plainspeak.co.za/styleguide

Reply:
Good idea, I have updated this. Thanks for the input (and the link to the styleguide, which I must admit I missed)